events-application-rest-api/src/main/java/com/bounswe/bounswe2017group3/Model/User.java

```diff
@@ -51,8 +51,19 @@
     /**
      * Deletion Date of the user.
      */
+    @Temporal(javax.persistence.TemporalType.DATE)
     private Date deletedAt;
     
+    
+    public User(){
```

Comment:
We can make it like `public User(){}` since the method body is empty 😄  I suggest to run an autoformat with your IDE, it should be sufficient.